<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_19">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_22" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_30">
      <item>
       <widget class="QWidget" name="widget_23" native="true">
        <property name="maximumSize">
         <size>
          <width>160</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background:rgb(229, 229, 229);
padding:5;</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_31">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label_78">
             <property name="styleSheet">
              <string notr="true">font: 12pt &quot;Microsoft YaHei UI&quot;;</string>
             </property>
             <property name="text">
              <string>当前项目信息</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_79">
             <property name="text">
              <string>项目名称:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="pro">
             <property name="text">
              <string>s</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_81">
             <property name="text">
              <string>运动员:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="userName">
             <property name="text">
              <string>user</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_83">
             <property name="text">
              <string>起始时间:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="sTime">
             <property name="text">
              <string>time</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_85">
             <property name="text">
              <string>总用时:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="uTime">
             <property name="text">
              <string>12&quot;85</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_87">
             <property name="text">
              <string>帧率:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_88">
             <property name="text">
              <string>30</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_89">
             <property name="text">
              <string>单位:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_90">
             <property name="text">
              <string>m</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_91">
             <property name="text">
              <string>备注:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextBrowser" name="textBrowser">
             <property name="styleSheet">
              <string notr="true">border: 0</string>
             </property>
             <property name="html">
              <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;啊啊啊啊啊啊啊啊&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_24" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_16">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="widget_25" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_32">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="t4">
              <property name="text">
               <string>基础数据</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="t5">
              <property name="text">
               <string>夹角数据</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="t6">
              <property name="text">
               <string>肢体平面夹角</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="t7">
              <property name="text">
               <string>关节投影角</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="t8">
              <property name="text">
               <string>平面投影角</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="t9">
              <property name="text">
               <string>技术指标</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="t10">
              <property name="text">
               <string>视频回放</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QStackedWidget" name="stackedWidget">
           <property name="currentIndex">
            <number>1</number>
           </property>
           <widget class="QWidget" name="page"/>
           <widget class="QWidget" name="page2">
            <layout class="QVBoxLayout" name="verticalLayout_20">
             <item>
              <widget class="QWidget" name="widget_26" native="true">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>30</height>
                </size>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_29">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QPushButton" name="contrastBtn">
                  <property name="text">
                   <string>对比项目</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="contrastLabel">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_8">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="details" native="true"/>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
